cfconfiguration/cffinder_unix.go

```diff
@@ -0,0 +1,50 @@
+// +build !windows
+
+package cfconfiguration
+
+import "go/build"
+import "os"
+import "os/user"
+
+var checkFileExistence = func (name string) (os.FileInfo, error) { return os.Stat(name) }
+
+func getUserConfigFile() (val string, ok bool) {
+	const relativeUserConfigPath = "/.config/cftool/config.yaml"
+
+	usr, err := user.Current()
+	if err != nil {
+		return "", false
+	}
+
+	userConfigPath := usr.HomeDir + relativeUserConfigPath
+
+	if _, err := checkFileExistence(userConfigPath); err != nil {
+		return "", false
+	}
+
+	return userConfigPath, true
+}
+
+func getGlobalConfigFile() (val string, ok bool) {
+	const globalConfigPath = "/etc/.Appliscale/cftool/config.yaml"
+
+	if _, err := checkFileExistence(globalConfigPath); err != nil {
+		return "", false
+	}
+
+	return globalConfigPath, true
+}
+
+func getConfigFileFromProjectRoot() (val string, ok bool) {
+	const relativeProjectRoot = "github.com/Appliscale/cftool"
```

Comment:
By project root I meant current directory of **your project**. :wink:  Imagine a situation when someone would like to have 2 repositories with _CF_ that has different settings:

- `~/projectA`
- `~/projectB`

The only way to differentiate settings is to put a file `.cftool` inside a repository (keep in mind the `.` in front).

In other words: `cftool` should pick up `.cftool` file from the current directory it is executed.